src/pages/patch/patchTabs/Util.tsx

```diff
@@ -0,0 +1,62 @@
+import { SorterResult } from "antd/es/table/interface";
+import {
+  TaskResult,
+  SortDirection,
+  SortOrder,
+  TaskSortCategory,
+} from "gql/generated/types";
+
+// takes sort input from the antd table and translates into part of the query string
+export const toSortString = (
+  sorts: SorterResult<TaskResult> | SorterResult<TaskResult>[]
+) => {
+  let sortStrings: string[] = [];
+  const shortenSortOrder = (order: string) =>
+    order === "ascend" ? SortDirection.Asc : SortDirection.Desc;
+  if (Array.isArray(sorts)) {
+    sorts.forEach((sort) => {
+      const singleSortString = `${sort.columnKey},${shortenSortOrder(
+        sort.order
+      )}`;
+      sortStrings = sortStrings.concat(singleSortString);
+    });
+  } else {
+    sortStrings = sortStrings.concat(
+      `${sorts.columnKey},${shortenSortOrder(sorts.order)}`
+    );
+  }
+
+  return sortStrings.join(";");
+};
+
+// takes a sort query string and parses it into valid GQL params
+export const parseSortString = (sortQuery: string | string[]): SortOrder[] => {
+  let sorts: SortOrder[] = [];
+  let sortArray: string[] = [];
+  if (typeof sortQuery === "string") {
+    sortArray = sortQuery.split(";");
+  } else {
+    sortArray = sortQuery;
+  }
+  if (sortArray?.length > 0) {
+    sortArray.forEach((singleSort) => {
+      const parts = singleSort.split(",");
```

Comment:
Out of curiousity why use `,` as the identifier for key value pairs? 
why not `:` since thats what is typically used.

Reply:
No special reason, I just picked some separator. Since this will be url encoded and not really human-readable, I don't feel that one is better than the other

Reply:
changed to :